Gemfile

```diff
@@ -61,6 +61,9 @@ group :development, :test do
   gem 'mock_redis', '~> 0.26'
   gem 'pre-commit', '~> 0.39'
   gem 'rspec-rails', '~> 4.0', '>= 4.0.1'
+  gem 'rubocop', require: false
```

Comment:
@danielvdao as a heads up, we might have to move these back out because of an issue with requiring Rubocop in the Rakefile causing a `LoadError` during deployment.